src/TypeScriptGenerator.ts

```diff
@@ -594,7 +594,9 @@ function getRefTypeName(name: string): string {
 }
 
 export const transforms: TypeGenerator["transforms"] = [
-  IRTransforms.commonTransforms[2], // RelayRelayDirectiveTransform.transform
-  IRTransforms.commonTransforms[3], // RelayMaskTransform.transform
-  IRTransforms.printTransforms[0] // FlattenTransform.transformWithOptions({})
-];
+  IRTransforms.commonTransforms[1], // RelayRelayDirectiveTransform.transform
+  IRTransforms.commonTransforms[2], // RelayMaskTransform.transform
+  IRTransforms.commonTransforms[3], // RelayMatchTransform.transform
+  IRTransforms.printTransforms[3], // FlattenTransform.transformWithOptions({})
+  IRTransforms.commonTransforms[4], // RelayRefetchableFragmentTransform.transform
```

Comment:
The relay compiler gets the transforms to apply from the Generator which (for typescript) is the file I changed.
I'm not sure if there's ever a case when the transforms should be different for typescript vs flow.. if not, probably the list should just reside in the relay compiler.
If there's a case for generator specifying it's own transforms then I think it would be best for relay compiler to export the transforms as part of its public API so this plugin doesn't have to rely on finding the transforms it needs at indexes in arrays that are likely to change again

Reply:
I don't think we gonna have different transforms for flow and ts

can you modify your pull request to get this transforms directly from relay-compiler in the same order?